pom.xml

```diff
@@ -124,7 +124,7 @@
     <parquet.version>1.11.1</parquet.version>
     <helix.version>1.0.4</helix.version>
     <zkclient.version>0.7</zkclient.version>
-    <jackson.version>2.12.7</jackson.version>
+    <jackson.version>2.12.7.1</jackson.version>
```

Comment:
this is version for `jackson-bom`. it should be changed to `2.12.7.20221012`
https://mvnrepository.com/artifact/com.fasterxml.jackson/jackson-bom

Reply:
Fixed.